domains/blake.json

```
@@ -0,0 +1,12 @@
+{
+	"description": "A redirection to my github page",
+	"repo": "https://github.com/BlakeH2208",
+	"owner": {
+		"username": "BlakeH2208",
+		"email": "contact@blakehurley.com"
+	},
+
+	"record": {
+		"A": "89.41.169.49"
```

`A` record has to be an array of ips

```suggestion
		"A": ["89.41.169.49"]
```